app.py

```diff
@@ -35,6 +35,44 @@ class Sitters(db.Model):
     zipCode = db.Column(db.Integer, nullable=False)
 
 
+class PetOwners(db.Model):
+    __tablename__ = "PetOwners"
+    id = db.Column(db.Integer, primary_key=True)
+    firstName = db.Column(db.String(256), nullable=False)
+    lastName = db.Column(db.String(256), nullable=False)
+    phoneNumber = db.Column(db.Integer, nullable=False)
+    streetAddress = db.Column(db.String(256), nullable=False)
+    city = db.Column(db.String(128), nullable=False)
+    state = db.Column(db.String(2), nullable=False)
+    zipCode = db.Column(db.Integer, nullable=False)
+    email = db.Column(db.String(256), nullable=False)
+    password = db.Column(db.String(256), nullable=False)
+    dogs = db.Relationship('Dogs')
+
+
+class DogSizes(db.Model):
+    __tablename__ = "DogSizes"
+    id = db.Column(db.Integer, primary_key=True)
+    size = db.Column(db.String(256), nullable=False)
```

Comment:
documentation names this attribute "sizeName", needs to be change either here or in documentation

Reply:
i think i'll change it to "name" so it will be DogSizes.name